server/src/routers/users.ts

```diff
@@ -12,7 +12,7 @@ import {
 export default function usersRouter(router: express.Router) {
   router.get("/users", getUsers);
   router.get("/users/:id", getUser);
-  router.post("/user", postUser);
+  router.post("/users", postUser);
   router.delete("/users/:id", deleteUser);
```

Comment:
no s

Reply:
Alright, thanks for pointing it out. Already fixed